plugins/server/io.github.cotrin8672/line-webhook/2.0/documentation.md

```diff
@@ -0,0 +1,23 @@
+# Line Webhook Signature Validation Plugin
```

Comment:
Please remove the top heading here - the project generator always uses the plugin name automatically.